src/components/TodoFooter/TodoFooter.tsx

```diff
@@ -0,0 +1,75 @@
+import classNames from 'classnames';
+import { Filter } from '../../types/FilterEnum';
+
+interface Props {
+  todoCounter: number;
+  filterTodos: Filter;
+  setFilterTodos: (filter: Filter) => void;
+}
+
+export const TodoFooter: React.FC<Props> = ({
+  todoCounter,
+  filterTodos,
+  setFilterTodos,
```

Comment:
props naming can be better, hints
- you already know you in `Todo`Footer so any `todo` prefixes or suffixes should have a good reason or it's just extra duplicating
- try to avoid names like you use for useState